app/cinema/hall.py

```diff
@@ -0,0 +1,19 @@
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self, movie_name: str, customers: list[Customer],
+                      cleaning_staff: Cleaner) -> None:

```

Comment:
Jeżeli nasza funkcja ma wiele argumentów, dobrą praktyką jest rozdzielanie ich na oddzielne linijki, aby zwiększyć czytelność kodu.

```suggestion
    def movie_session(
        self, 
        movie_name: str, 
        customers: list[Customer],
        cleaning_staff: Cleaner
    ) -> None:

```